### ISSUE TITLE
bug: wrong operation numbering!!!

### ISSUE THREAD

**Issue by kkafar** [kkafar/master-monorepo#223]
## Bug description

Make sure that my method of parsing instances is compliant with following description:

```
Operation numbering
The operations are numbered as follows: 
The first n operations refer to the first operation of each job (according to order of the jobs),
operations n+1,...,2n regard the second operation of the n jobs, and so on.
So operation i is the k'th operation of job j, where k = ceil(i/n) and j = i mod n.
```

### Discussed in https://github.com/kkafar/master-monorepo/discussions/277

<div type='discussions-op-text'>

<sup>Originally posted by **kkafar** May  11, 2024</sup>

## Implementation progress

* #278
* #279 
* #280 
* #281 
* #282 
* #283
* #285

**Comment by kkafar** [comment]
Yeah, I'm using wrong operation numbering for sure. However right now I'm not verifying whether I've found new solution or not, so I do not have to fix this now.

**Comment by kkafar** [comment]
Closed with #287
